docs/networking.md

```diff
@@ -9,9 +9,14 @@ help you out.
 
 
 {% include /style/icon.html type="info" %}
-For Raspberry Pi platforms with no display, you must use the wired Ethernet port
-to connect for the first time. You can set up additional connections using the
-`connmanctl` command through a remote SSH session.
+For Raspberry Pi platforms with no display 
+([to be able to use Brickman](http://lechnology.com/2016/05/adding-a-display-to-brickpi/)), 
```

Comment:
This parenthetical is confusing IMHO. In general, I'd prefer to keep the amount of text down in these alerts; if we need a lot of text, we can switch this to a panel.